app/views/calls/_new_call.html.erb

```diff
@@ -6,6 +6,15 @@
           <button type="button" class="close close-terms" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
           <h4 class="modal-title calls-request">Call <b><%= p.patient.name %></b> now:</h4>
           <br>
+          <% if !p.patient.other_contact.blank? && !p.patient.other_phone.blank? %>
+          <h4 class="modal-title calls-request calls-other-contact">
+            <%= p.patient.other_contact %>
+            <% if !p.patient.other_contact_relationship.blank? %> (<%= p.patient.other_contact_relationship %>) <% end %>
```

Comment:
rails-y style feedback -- the standard style guide (which we follow only loosely, but still) tends to favor positive truthy checks. You might think about rewriting this to use `present?` if you've got a sec, though it'll def be fine either way. 

(Also it's a nice opportunity to check out some of the stuff buried in activesupport -- http://api.rubyonrails.org/classes/Object.html#method-i-present-3F )